TypeScript/TS/tscript.ts

```diff
@@ -0,0 +1,382 @@
+var buttonText:string|number, tempVarTrigo:string|number;
+var result = document.querySelector("#result")! as HTMLInputElement;
+var resultToShow = document.querySelector(".result")! as HTMLInputElement;
+var resultValue:string|number = "";
+var buttons= document.querySelectorAll("button")! ;
+var buttonLength = document.querySelectorAll("button").length;
+var degButton = document.querySelector(".deg")!;
+var radButton = document.querySelector(".rad")!;
+var memorySave:number|string;
```

Comment:
type `number | string` is used more than once in the file.
Create a type alias for same and replace it everywhere.